recipes/_rhel_scheduler.rb

```diff
@@ -0,0 +1,37 @@
+# Do some stuff RHEL specific
+include_recipe 'java' if node['aurora']['install_java']
+
+# Add Aurora repository
+include_recipe "aurora::_rhel_repo"
```

Comment:
I don't understand why you are installing mesos without using the mesos cookbook?